*   Проект доделывал на Windows 10, Python 3.10
**  В папке dist_wheel то, что получилось при использовании wheel
*** В папке dist_pyinstaller то, что получилось при использовании pyinstaller

1.  При создании дистрибутива через python sdist bdist_wheel не все папки и файлы загружаются в дистрибутив.
    Пришлось записывать вручную.
2.  При попытке залить на PyPi через python setup.py upload выдает ошибку:
    Upload failed (400): Invalid value for blake2_256_digest.
3.  При создании дистрибутива через pyinstaller:
    pyinstaller --onefile --add-data "client_dist;." client.py (аналогично для сервера)
    создает екзешник, который ничего не запускает.

* В базе приложения записаны два клиента client1 и client2 с паролями 123456.
Все возникавшие ошибки отследил, но с отправкой сообщений не вижу, в каком направлении искать.
4.  При запуске приложения из консоли при отправке сообщения от одного клиента другому отправляющий клиент
    получает ошибку сервера и отключается.
    Локализовать проблему не могу.
    Сервер обрабатывает полученное сообщение в стр.98 server_start (функция process_message отрабатывает),
    после этого возникает исключение в стр.92 server_start "Ошибка при получении сообщения".
    Как отследить причину ошибки, не знаю.